frontend/src/core/comments/components/AddComment.js

```diff
@@ -47,28 +47,26 @@ export default function AddComments(props: Props) {
         }
     }
 
-    const submitComment = (event: SyntheticKeyboardEvent<>) => {
-        if (event.keyCode === 13 && event.shiftKey === false) {
-            event.preventDefault();
-            const comment = commentInput.current.value;
+    const submitComment = (event: SyntheticMouseEvent<>) => {
+        event.preventDefault();
+        const comment = commentInput.current.value;
 
-            if (!comment) {
-                return null;
-            }
+        if (!comment) {
+            return null;
+        }
 
-            addComment(comment, translation);
+        addComment(comment, translation);
 
-            commentInput.current.value = '';
-            commentInput.current.rows = minRows;
-        }
+        commentInput.current.value = '';
+        commentInput.current.rows = minRows;
     };
 
     return <div className='comment add-comment'>
         <UserAvatar
             username={ username }
             imageUrl={ imageURL }
         />
-        <form className='container'>
+        <form className='form-container'>
```

Comment:
I'd leave the `.container` class here, and then just override 1 differing CSS rule (`flex-flow`) with a more specific selector, e.g. `comments-list .comment.add-comment .container`.